src/Castle.Core.Tests/DynamicProxy.Tests/ComInterfacesTests.cs

```diff
@@ -42,40 +42,18 @@ public void Marshal_Release_throws_when_called_with_IntPtr_Zero()
 		// how the tests should run, e.g. using the `[Platform]` attributes.
 
 		[Test]
-		[Platform(Include = "Win", Reason = "Depends on ADODB.dll, which can fail to load under Mono/Linux.")]
 		public void Can_proxy_ADO_RecorsSet()
 		{
-			Implementation.Can_proxy_ADO_RecorsSet(generator);
+			generator.CreateInterfaceProxyWithoutTarget<Recordset>();
 		}
 
 		[Test]
 		[Platform(Include = "Win", Reason = "Depends on OLE32.dll due to co-class instantiation, which is likely only available on Windows.")]
 		public void Can_proxy_existing_com_object()
 		{
-			Implementation.Can_proxy_existing_com_object(generator);
-		}
-
-		private static class Implementation
-		{
-			public static void Can_proxy_ADO_RecorsSet(ProxyGenerator generator)
-			{
-				generator.CreateInterfaceProxyWithoutTarget<Recordset>();
-			}
-
-			public static void Can_proxy_existing_com_object(ProxyGenerator generator)
-			{
-				var command = new Command();
-				var parameter = ExtractActualComParameterObject(command.CreateParameter("foo"));
-				generator.CreateInterfaceProxyWithTargetInterface(parameter);
-			}
-
-			private static Parameter ExtractActualComParameterObject(Parameter parameter)
```

Comment:
This is no longer needed, because there will no longer be any custom marshaler converting the actual `Parameter` COM object to an `InternalParameter` wrapper.